### ISSUE TITLE
theme: simplify linking to other page's images

### ISSUE THREAD

**Issue by kultpro** [McShelby/hugo-theme-relearn#406]
How is it optimal to specify relative links to images?

Now only the link of the type "/works finecat1/ca1.2/images.jpg". But if the catalog is renamed, will have to check every article to edit all the links.

The name of the image "images.jpg " - good for sections. On a separate page, you have to specify "../images.jpg ". However, the image is not displayed when printing the page until there is a link of the type "/cat1/ca1.2/images.jpg ".

**Comment by McShelby** [comment]
Your structure in the `content` directory is unclear, so it's hard to understand your issue. Please link a repo or post your file/directory strucutre here.

Usually images in `content` are only linkable from their containing branch bundle. You may have to read about that in the Hugo documentation.

Nevertheless, it also bothers me, to not be able to link resources from other branch bundles. I am currently working on a patch, but it is far from done and I can't promise when or if it may land in the release.

**Comment by kultpro** [comment]
Thanks for the reply.

1. "Web files location"
2. "Image location"
3. "Working code for the image"
---
1. `/catalog/_index.md`
2. `/catalog/images.jpg`
3. `<img src="images.jpg"`
3. `<img src="/catalog/images.jpg"`
---
1. `/catalog/article.md`
2. `/catalog/images.jpg`
3. `<img src="../images.jpg"`
3. `<img src="/catalog/images.jpg"`
---
index.print.html displays images only with the type code "/catalog/images.jpg ". This is probably some kind of mistake. Here's what the source code of the page shows if the path to the image is specified as "../images.jpg"

`[<img src="../../catalog/index.html/../catalog/images.jpg?classes=shadow"]`

I would like in both cases the same type of indication of the path to the image. And error-free display of images on the print page. 

Thank you so much for your feedback and your work!

**Comment by McShelby** [comment]
In case of Hugo, this makes no sense:

````
  /catalog/article.md
  /catalog/images.jpg
  <img src="../images.jpg"
  <img src="/catalog/images.jpg"
````

Your image is "orphaned" because it does not belong to a branch bundle. A agree with you, that this is non intuitive but thats the way Hugo works.

**Comment by McShelby** [comment]
Should be addressed together with #377 for completeness

**Comment by McShelby** [comment]
Further discussion in #587. Feel free to leave your comments there.
